pkgs/applications/video/mpv/scripts/memo.nix

```
@@ -0,0 +1,38 @@
+{ fetchFromGitHub
+, buildLua
+, lib
+, unstableGitUpdater
+}:
+
+buildLua {
+  pname = "memo";
+  version = "0-unstable-2023-10-31";
+
+  src = fetchFromGitHub {
+    owner = "po5";
+    repo = "memo";
+    rev = "6f2b036ef860e130ea584657389e0a7ceb4d954f";
+    hash = "sha256-m8ikXuw7PM4Btg8w7ufLneKA4fnYjMyfVJYueZILMw8=";
+  };
+  passthru.updateScript = unstableGitUpdater {};
+
+  passthru.scriptName = "memo.lua";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/mpv/scripts
+    cp -r memo.lua $out/share/mpv/scripts
```

Please use `install -Dt`.

---

it seems like `installPhase` and `passthru.scriptName` is completely redundant.
```suggestion
  passthru.scriptName = "memo.lua";

  installPhase = ''
    runHook preInstall

      install -Dt memo.lua $out/share/mpv/scripts

    runHook postInstall
  '';
```

---

Even `passthru.scriptName` is not needed because if will be computed from `pname`.

---

removed both :+1: